files/build_templates/sonic_debian_extension.j2

```diff
@@ -182,6 +184,17 @@ sudo LANG=C chroot $FILESYSTEM_ROOT systemctl enable hostcfgd.service
 sudo cp $IMAGE_CONFIGS/hostcfgd/hostcfgd $FILESYSTEM_ROOT/usr/bin/
 sudo cp $IMAGE_CONFIGS/hostcfgd/*.j2 $FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES/
 
+# copy core file uploader files
+sudo cp $IMAGE_CONFIGS/corefile_uploader/core_uploader.service $FILESYSTEM_ROOT/etc/systemd/system/
```

Comment:
it would be better to put all these into sonic-utilities. In the build image repo, we just need to install the core_uploader service.

Reply:
ok.

Reply:
I leave core_uploader.py & its rc file here, as they are part of .service.
Moved update_json.py to sonic-utilities.